src/Components/App/App.js

```diff
@@ -37,19 +42,26 @@ class App extends Component {
     this.setState({wordList: removeTile})
   }
 
-  //save method (need to pass into MainContainer)
+  saveTile = (id) => {
+    const saveCard = this.state.wordList.filter(card => card.id === id).flat()
+    this.setState({savedWords: [...this.state.savedWords, saveCard[0]]})
```

Comment:
would a find be better here if you're just looking for one? Just a suggestion but not super necessary